packages/react-renderer-demo/src/layout.js

```diff
@@ -105,9 +105,11 @@ const useStyles = makeStyles(theme => ({
 const Layout = ({ children, location: { pathname }}) => {
   const classes = useStyles();
   const [ open, setOpen ] = React.useState(false);
+  const searchRef = React.useRef(null);
 
   function handleDrawerOpen() {
     setOpen(true);
+    setTimeout(() => searchRef.current.focus(), 500);
```

Comment:
Why the timeout? Was there some issue with immediate focus?

Reply:
Yes